PerimeterXModule/Internals/Helpers/PxCommonUtils.cs

```diff
@@ -68,5 +71,43 @@ public static void AddHeaderToRequest(HttpContext context, string key, string va
 				ro.SetValue(headers, true, null);
 			}
 		}
+
+		public static bool IsEmailAddress(string str)
```

Comment:
Fix indentation in this file

Reply:
Everything looks fine in the source file (and not on Github)